setup.cfg

```diff
@@ -1,8 +1,39 @@
-[bdist_wheel]
-universal = 1
-
 [metadata]
-license_file = LICENSE
+name = django-ecsmanage
```

Comment:
Moving these settings from `setup.py` to `setup.cfg` allows for a more declarative, config file-style approach. (azavea/django-amazon-ses#56)